Helper/Ratings.php

```diff
@@ -0,0 +1,213 @@
+ *
+ * @author Nosto Solutions Ltd <contact@nosto.com>
+ * @copyright 2017 Nosto Solutions Ltd
+ * @license http://opensource.org/licenses/BSD-3-Clause BSD 3-Clause
+ *
+ */
+
+namespace Nosto\Tagging\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Catalog\Model\Product;
+use Nosto\Tagging\Helper\Data as NostoHelperData;
+use Magento\Store\Model\Store;
+use Nosto\Tagging\Logger\Logger as NostoLogger;
+use Magento\Review\Model\ReviewFactory;
+use Nosto\Tagging\Model\Product\Ratings as ProductRatings;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Module\Manager;
+
+/**
+ * Price helper used for product price related tasks.
+ */
+class Ratings extends AbstractHelper
+{
+    const REVIEW_COUNT= 'reviews_count';
+    const AVERAGE_SCORE = 'average_score';
+
+    private $moduleManager;
+    private $nostoDataHelper;
+    private $logger;
+    private $reviewFactory;
+    private $ratingsFactory;
+
+    /**
+     * Ratings constructor.
+     * @param Context $context
+     * @param Manager $moduleManager
+     * @param Data $nostoHelperData
+     * @param ReviewFactory $reviewFactory
+     * @param NostoLogger $logger
+     * @param RatingsFactory $ratingsFactory
+     */
+    public function __construct(
+        Context $context,
+        Manager $moduleManager,
+        NostoHelperData $nostoHelperData,
+        ReviewFactory $reviewFactory,
+        NostoLogger $logger,
+        RatingsFactory $ratingsFactory
+    ) {
+        parent::__construct($context);
+        $this->moduleManager = $moduleManager;
+        $this->nostoDataHelper = $nostoHelperData;
+        $this->logger = $logger;
+        $this->reviewFactory = $reviewFactory;
+        $this->ratingsFactory = $ratingsFactory;
+    }
+
+    /**
+     * Get ratings
+     *
+     * @param Product $product
+     * @param Store $store
+     * @return ProductRatings|null
+     */
+    public function getRatings(Product $product, Store $store)
+    {
+        $ratings = $this->getRatingsFromProviders($product, $store);
+        if ($ratings === null) {
+            return null;
+        }
+
+        $productRatings = new ProductRatings();
+        $productRatings->setReviewCount($ratings[self::REVIEW_COUNT]);
+        $productRatings->setRating($ratings[self::AVERAGE_SCORE]);
+        return $productRatings;
+    }
+
+    /**
+     * Get Ratings of product from different providers
+     *
+     * @param Product $product
+     * @param Store $store
+     * @return array|null
+     */
+    private function getRatingsFromProviders(Product $product, Store $store)
+    {
+        if ($this->nostoDataHelper->isRatingTaggingEnabled($store)) {
+            $provider = $this->nostoDataHelper->getRatingTaggingProvider($store);
+
+            if ($provider === NostoHelperData::SETTING_VALUE_YOTPO_RATINGS) {
+                if (!$this->moduleManager->isEnabled("Yotpo_Yotpo")) {
```

Comment:
Use `$this->canUseYotpo()` here.

```suggestion
                if (!$this->canUseYotpo()) {
```